h-youjin/LV.1/행렬의-덧셈.js

```diff
@@ -0,0 +1,12 @@
+function solution(arr1, arr2) {
+  const answer = [];
+
+  for (let i = 0; i < arr1.length; i++) {
+    const sum = [];
+    for (let j = 0; j < arr1[i].length; j++) {
+      sum.push(arr1[i][j] + arr2[i][j]);
+    }
+    answer.push(sum);
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(arr1, arr2) {
    const answer = [[]];
    for(let i = 0; i < arr1.length; i ++) {
        answer[i] = []
        for(let j = 0; j < arr1[i].length; j++) {
            answer[i].push(arr1[i][j] + arr2[i][j])
        }
    }
    return answer;
}
```
이렇게 answer 안에 빈 배열을 하나 더 둠으로써, 변수를 하나만 선언하고 해결할 수도 있습니다! for 문을 활용해서 잘 해결하셨네요👍

Reply:
감사합니다 대운님👍 해당 방법 활용해서 적용해보겠습니다~